extensions/amp-story/1.0/amp-story.js

```diff
@@ -1022,11 +1022,12 @@ export class AmpStory extends AMP.BaseElement {
         {tagOrTarget: 'AMP-LIVE-LIST', method: 'update'},
       ]);
 
-      this.element.addEventListener(AmpEvents.DOM_UPDATE, ({target}) => {
-        this.liveStoryManager_.update(
-          target,
-          this.element.querySelectorAll('amp-story-page:not([ad])')
-        );
+      this.element.addEventListener(AmpEvents.DOM_UPDATE, () => {
```

Comment:
Just curious: what else could trigger this event?

Reply:
Some other extensions in AMP use it, none of which we use now. I don't think there's a way of checking if it comes specifically from live-list though :( Do you prefer if I create another custom event or something? For now we just check that the story has the `live-story` attribute before adding this listener.